lib/main.dart

```diff
@@ -1,9 +1,14 @@
+import 'dart:html';
+
 import 'package:flame/game.dart';
 import 'package:flutter/material.dart';
 
 import 'heeve_game.dart';
 
 void main() {
+  // Prevent default event handler
+  document.onContextMenu.listen((event) => event.preventDefault());
```

Comment:
Is this to prevent right clicks?

Reply:
to prevent the context menu to show up, allowing right clicks